backend/database_handler/transactions_processor.py

```diff
@@ -149,9 +161,24 @@ def set_transaction_result(self, transaction_hash: str, consensus_data: dict):
         transaction.status = TransactionStatus.FINALIZED
         transaction.consensus_data = consensus_data
 
-        print(
-            "Updating transaction status",
-            transaction_hash,
-            TransactionStatus.FINALIZED.value,
-        )
         self.session.commit()
+        self.dispatch_transaction_status_update(
+            transaction_hash, TransactionStatus.FINALIZED
+        )
+
+    def dispatch_transaction_status_update(
```

Comment:
@denishacquin I'm not 100% sure that this should be here. I think it would be better not to have the msg_handler injected into the transactions_processor and log the transaction transitions in the consensus. 
What do you think @AgustinRamiroDiaz?

Reply:
makes sense, moved it to consensus and refactored the actual tx update together with it to reduce repetition but might be overkill - let me know what you think